components/servo/webview_delegate.rs

```diff
@@ -963,6 +963,11 @@ pub trait WebViewDelegate {
 
     /// Request to display a notification.
     fn show_notification(&self, _webview: WebView, _notification: Notification) {}
+
+    /// A console message was logged by content in this [`WebView`].
+    fn show_console_message(&self, _webview: WebView, level: ConsoleLogLevel, message: String) {
+        log::log!(level.into(), "{message}");
+    }
```

Comment:
Let's just do nothing here as this may interfere with operation of some embedder that prints to stdout in an unexpected way.